class/defaults.yml

```diff
@@ -14,4 +14,8 @@ parameters:
       kubectl:
         repository: docker.io
         image: ongres/kubectl
-        tag: v1.24.3-build-6.16
+        tag:
```

Comment:
Nitpick: The empty tag looks a bit weird, I'd probably either remove the tag completely or set it to empty string